elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -294,33 +322,30 @@ def render(element_html, data):
             except FileNotFoundError:
                 return 'The reference solution is not provided for this question.'
 
-        grading_mode = pl.get_string_attrib(element, 'grading-method', 'ordered')
-        if grading_mode == 'unordered':
-            grading_mode = 'in any order'
-        elif grading_mode == 'dag' or grading_mode == 'ranking':
-            grading_mode = 'one possible correct order'
+        if grading_method == 'unordered':
+            ordering_message = 'in any order'
+        elif grading_method == 'dag' or grading_method == 'ranking':
+            ordering_message = 'one possible correct order'
         else:
-            grading_mode = 'in the specified order'
+            ordering_message = 'in the specified order'
         check_indentation = pl.get_boolean_attrib(element, 'indentation', INDENTION_DEFAULT)
         indentation_message = ', with correct indentation' if check_indentation is True else None
 
-        if answer_name in data['correct_answers']:
```

Comment:
I couldn't figure out a good reason for this check to be here (it isn't in any of the other elements).